### PR TITLE
fix: update account creation card

### PR DESCRIPTION
**Pull request by pete-watters**
This PR makes some changes to the account creation card and introduces a new intermediary drawer screen:

https://github.com/user-attachments/assets/bfbe8b35-d6b5-4187-8415-3fe554f2a7d1

### CODE REVIEW

File: apps/mobile/src/components/widgets/accounts/sheets/add-account-sheet.layout.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      ref={addAccountSheetRef}
+      themeVariant={themeVariant}
+    >
+      <AnimatedBox style={animatedStyle}>
+        <Box p="5">
+          <Text pb="5" variant="heading05">
+            {t`Add account`}
+          </Text>
+          <Box flexDirection="column" gap="1">
+            <AddWalletListItem
```
Comment (pete-watters) [inline apps/mobile/src/components/widgets/accounts/sheets/add-account-sheet.layout.tsx:61]: I copied this from the `add-wallet` sheet. I think I should refactor `AddWalletListItem` to share it. 

Maybe to a different component called `ListItem`? I was trying to think of a good name / place for it

**Review by kyranjamie — APPROVED** [review]

**Review by fbwoolf — APPROVED** [review]

File: packages/ui/src/icons/wallet-plus-icon.native.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import WalletPlus from '../assets/icons/wallet-plus-24-24.svg';
+import { Icon, IconProps } from './icon/icon.native';
+
+export function WalletPlusIcon({ variant, ...props }: IconProps) {
```
[the remark this answers is not in the document]
Comment (pete-watters) [reply packages/ui/src/icons/wallet-plus-icon.native.tsx:4]: Apologies, I forgot that. I'll update.

I couldn't find the icon list at first

File: apps/mobile/src/components/widgets/accounts/sheets/add-account-sheet.layout.tsx (hunk 1 above)
Comment (pete-watters) [reply apps/mobile/src/components/widgets/accounts/sheets/add-account-sheet.layout.tsx:61]: Sounds good, thanks! Naming can be hard sometimes. 

I'll refactor them in https://github.com/leather-io/issues/issues/138 

I can also update to migrate my `TouchableOpacity` code to `pressable`

File: packages/ui/src/icons/wallet-plus-icon.native.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (pete-watters) [reply packages/ui/src/icons/wallet-plus-icon.native.tsx:5]: I tried to do this quickly but made a mess, will fix it ASAP. 

More Haste, Less Speed

**Pull request merged**